tests/automation/message_checks_groups.spec.ts

```diff
@@ -18,302 +14,250 @@ import {
+      replyText: testReply,
+      receiverWindow: aliceWindow1,
+    });
+  },
+);
 
-test('Send voice message to group', async () => {
-  const [windowA, windowB, windowC] = await openApp(3);
-  const [userA, userB, userC] = await Promise.all([
-    newUser(windowA, 'Alice'),
-    newUser(windowB, 'Bob'),
-    newUser(windowC, 'Charlie'),
-  ]);
-  const group = await createGroup(
-    'Message checks',
-    userA,
-    windowA,
-    userB,
-    windowB,
-    userC,
-    windowC,
-  );
-  const testReply = `${userB.userName} replying to voice message from ${userA.userName} in ${group.userName}`;
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'microphone-button',
-  });
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'session-toast',
-  });
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'enable-microphone',
-  });
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'message-section',
-  });
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'microphone-button',
-  });
-  await sleepFor(5000);
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'end-voice-message',
-  });
-  await sleepFor(2000);
-  await clickOnElement({
-    window: windowA,
-    strategy: 'data-testid',
-    selector: 'send-message-button',
-  });
-  await sleepFor(1000);
-  await lookForPartialTestId(windowB, 'audio-', true, true);
-  await lookForPartialTestId(windowC, 'audio-');
-  await clickOnMatchingText(windowB, 'Reply');
-  await sendMessage(windowB, testReply);
-  await waitForTextMessage(windowA, testReply);
-});
+test_group_Alice_1W_Bob_1W_Charlie_1W(
+  'Send voice message to group',
+  async ({
+    alice,
+    bob,
+    aliceWindow1,
+    bobWindow1,
+    charlieWindow1,
+    groupCreated,
+  }) => {
+    const testReply = `${bob.userName} replying to voice message from ${alice.userName} in ${groupCreated.userName}`;
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'microphone-button',
+    });
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'session-toast',
+    });
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'enable-microphone',
+    });
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'message-section',
+    });
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'microphone-button',
+    });
+    await sleepFor(5000);
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'end-voice-message',
+    });
+    await sleepFor(2000);
+    await clickOnElement({
```

Comment:
Would be one day good to setup a functions that can support multiple sequential clicks and you just pass the selectors in as an array

Reply:
eventually, yeah